jaxley/modules/base.py

```diff
@@ -179,7 +183,7 @@ def _childviews(self) -> List[str]:
     def __getitem__(self, index):
         supported_lvls = ["network", "cell", "branch"]  # cannot index into comp
 
-        # TODO: SHOULD WE ALLOW GROUPVIEW TO BE INDEXED?
+        # TODO FROM #447: SHOULD WE ALLOW GROUPVIEW TO BE INDEXED?
```

Comment:
The problem is that for Modules, i.e. ´net[0,0,0]` it is straightforward to lazy index into by just calling `net.cell(0).branch(0).comp(0)` internally, likewise for `net.cell(0)[0,0]`. However, for `net.cell0` this is not the case, since we cannot tell easily what the user is indexing into. I.e. they might do `net.cell0` or `net.comp0`, which are totally different, but equivalent on the group level. Now should we allow them to still be able to do `net.comp0[0,0,0]`? 

Decided to disable it for now and only allow indexing into Modules and Views thereof. I think that is sensible. The user can of course still index into groups however they like, they just have to do it via `comp`, `cell`, `branch` instead of via `__getitem__`.